### PR TITLE
Create draft-integration.md

### PR DESCRIPTION
**Pull request by TuanaCelik**
I'm creating this draft integration file that can be re-used which should hopefully help us make these pages more uniform.

### CODE REVIEW

File: draft-integration.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+layout: integration
+name: Name of your integration
+description: Description of integration
+authors:
+    - name: author-name
+      socials:
+        github: author-username
+        twitter: author-username
+        linkedin: author-username
```
Comment (bilgeyucel) [inline draft-integration.md:10]: This should be the full path
```suggestion
        linkedin: author-linkedin-url
```

File: draft-integration.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+pypi: https://pypi.org/project/{project-slug}
+repo: https://your-repo-url
+type: Type of integration
```
Comment (bilgeyucel) [inline draft-integration.md:13]: Let's also give them some options to start with: "Model Provider", "Document Store"

File: draft-integration.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+## Installation
+
+```bash
+pip install your-package
+```
+
+## Usage
+### Components
+This instegration introduces X components:
```
Comment (bilgeyucel) [inline draft-integration.md:35]: Small typo
```suggestion
This integration introduces X components:
```

File: draft-integration.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+- The `ComponentName`: 
+  
+### Use XYZ
+
+```python
+Example code block
```
Comment (annthurium) [inline draft-integration.md:42]:
```suggestion
Example code block (must actually run, include all imports, etc.)
```

**Review by annthurium — APPROVED** [review]
THANK YOU

I left one small suggestion, feel free to take or leave it :D

File: draft-integration.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+toc: true
+---
+### **Table of Contents**
+- [Overview](#overview)
+- [Installation](#installation)
+- [Usage](#usage)
+- [License](#license)
+
+## Overview
+
```
Comment (bilgeyucel) [inline draft-integration.md:26]: Let's guide people more here
```suggestion
## Overview
Add more information about this integration such as its features, links to its documentation 

```

**Review by bilgeyucel — COMMENTED** [review]
Thank you for doing this @TuanaCelik 🙌

File: draft-integration.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+description: Description of integration
+authors:
+    - name: author-name
+      socials:
+        github: author-username
+        twitter: author-username
+        linkedin: author-linkedin-url
+pypi: https://pypi.org/project/{project-slug}
+repo: https://your-repo-url
+type: Type of integration
```
Comment (TuanaCelik) [inline draft-integration.md:13]:
```suggestion
type: Type of integration (e.g.: Model Provider or Document Sotre)
```

File: draft-integration.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+description: Description of integration
+authors:
+    - name: author-name
+      socials:
+        github: author-username
+        twitter: author-username
+        linkedin: author-linkedin-url
+pypi: https://pypi.org/project/{project-slug}
+repo: https://your-repo-url
+type: Type of integration (e.g.: Model Provider or Document Sotre)
```
Comment (TuanaCelik) [inline draft-integration.md:13]:
```suggestion
type: Type of integration (e.g.: Model Provider or Document Store)
```

**Pull request merged**